config/v1_0/fcos.go

```diff
@@ -83,7 +83,7 @@ func TranslateBytes(input []byte, options common.TranslateOptions) ([]byte, erro
 	second := validate.Validate(final, "json")
 	second.Correlate(translatedTree)
 	r.Merge(second)
-	fmt.Println(r.String())
```

Comment:
You hit this specific one, however we use this idiom pretty much everywhere.
I think it's more effective if we stuck a `strings.TrimSpace()` either in the `String` method or in `Correlate`/`Merge`.

Reply:
> I think it's more effective if we stuck a strings.TrimSpace() either in the String method or in Correlate/Merge

Hmm, not sure I follow. The unconditional newline comes from the `Println()` here, not `r.String()`.

Reply:
Ah, my understanding was that you got two newlines because "The `String()` method already adds a newline at the end of each report entry."